pom.xml

```diff
@@ -0,0 +1,168 @@
+                <version>4.13.2</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>2.0.13</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-slf4j2-impl</artifactId>
+                <version>2.23.1</version>
+                <!-- <scope>test</scope> Commented to enable logging-->
+            </dependency>
+
+
+        </dependencies>
+    </dependencyManagement>
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-source-plugin</artifactId>
+                    <version>3.3.1</version>
+                    <executions>
+                        <execution>
+                            <id>attach-sources</id>
+                            <goals>
+                                <goal>jar-no-fork</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-javadoc-plugin</artifactId>
+                    <version>3.8.0</version>
+                    <configuration>
+                        <quiet>true</quiet>
+                        <doclint>none</doclint>
+                    </configuration>
+                    <executions>
+                        <execution>
+                            <id>attach-javadocs</id>
+                            <phase>package</phase>
+                            <goals>
+                                <goal>jar</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
+
+    <developers>
+        <developer>
+            <name>Cristian Augusto</name>
+            <url>https://www.augustocristian.es</url>
+        </developer>
+        <developer>
+            <name>Claudio de la Riva</name>
+            <url>http://giis.uniovi.es</url>
+        </developer>
+        <developer>
+            <name>Javier Tuya</name>
+            <url>http://giis.uniovi.es</url>
+        </developer>
+    </developers>
+    <licenses>
+        <license>
+            <name>Apache-2.0</name>
+            <url>https://github.com/giis-uniovi/retorch/blob/main/LICENSE</url>
+        </license>
+    </licenses>
+    <scm>
+        <url>https://github.com/giis-uniovi/retorch</url>
+    </scm>
+
+    <!-- To publish packages (mvn deploy) a profile must be specified (GitHub for snapshots and maven central for releases) -->
+    <profiles>
```

Comment:
This repo will not generate artifacts for maven, remove these unnecessary parts.